Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:16 AS frontend
+
+WORKDIR /frontend
+COPY package*.json /frontend
```

Comment:
add `/` at the end of the line

Reply:
No problem
Does it work for you when you added / at the end of line 4?

I'm running Mac and both worked for some reason (with and without /)

Reply:
I tried adding `/` at the end of line 4 and it works!

Reply:
Cool, interesting!